bitrise.yml

```diff
@@ -265,7 +265,7 @@ workflows:
           inputs:
             - project_path: dev-app/ios/StripeTerminalReactNativeDevApp.xcworkspace
             - scheme: UnitTests
-            - destination: platform=iOS Simulator,name=iPhone 14,OS=latest
+            - destination: platform=iOS Simulator,name=iPhone 15,OS=17.2
```

Comment:
was this failing because no iPhone 14 simulator found?

you can also use

```
    - xcode-start-simulator@0:
       inputs:
       - destination: platform=iOS Simulator,name=iPhone 14,OS=latest
```

as a step prior to `xcode-test` to ensure the simulator exists...though I would expect the xcode-15 stack to still provide the same simulators 🤔 🤷‍♂️

Reply:
We tried that here https://app.bitrise.io/build/b9927dd4-5812-4f13-81fa-1b53d1fedf3a?tab=log and got 

```
Destination simulator:
Name: Bitrise iOS default
Device type: iPhone 8
OS: iOS Simulator 16.4
UDID: D577766E-7E37-4E39-AD1C-91A7699B8EA4
Status: Booted
```

and still failed with `Unable to find a destination matching the provided destination specifier`